.jenkins/pytorch/test.sh

```diff
@@ -173,10 +173,10 @@ test_aten() {
       SUDO=sudo
     fi
 
-    ${SUDO} ln -s "$TORCH_LIB_PATH"/libc10* build/bin
-    ${SUDO} ln -s "$TORCH_LIB_PATH"/libcaffe2* build/bin
-    ${SUDO} ln -s "$TORCH_LIB_PATH"/libmkldnn* build/bin
-    ${SUDO} ln -s "$TORCH_LIB_PATH"/libnccl* build/bin
+    ${SUDO} ln -s "$TORCH_LIB_PATH"/libc10* build/bin || true
```

Comment:
Perhaps we should just force an overwrite rather than fail to create a symlink?
```suggestion
    ${SUDO} ln -sf "$TORCH_LIB_PATH"/libc10* build/bin
```

Reply:
I thought about that, but @walterddr said that it's "unnecessary to symlink" in this case, which makes it sound like overwriting would be wrong; @walterddr could you comment on this? I really don't know what's going on here haha

Reply:
even better solution

Reply:
yup. I think we can just overwrite it. regardless what's already in the folder.

Reply:
ok I'll do that